src/main/java/core/basesyntax/strategy/BalanceOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    private final Storage storage;

```

Comment:
According to the encapsulation principle, the storage should not be exposed directly. Instead, consider using an interface for the storage operations and injecting it here. This would make your code more flexible and easier to test.